### PR TITLE
added script for deploying docs; closes #146

### LINKED ISSUE #146: Add deploy docs script

### PR DESCRIPTION
**Pull request by thuey**

### CODE REVIEW

File: scripts/deployDocs.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+mkdocs build
+
+if [ ! -d "pleaseops.github.io" ]; then
+  git clone git@github.com:pleaseops/pleaseops.github.io.git
+fi
+
+cd pleaseops.github.io
+git pull
+
+cp -R ../site/* ./docs
```
Comment (jonnymbgood) [inline scripts/deployDocs.sh]: Don't we also need to copy over `../www/*`?

**Comment by jonnymbgood** [comment]
@thuey Also, you do need to delete everything in the repo and re-copy. If something gets deleted in the source directory it won't get removed when you copy. Or we can use a sync style command.

File: scripts/deployDocs.sh (hunk 1 above)
Comment (thuey) [reply scripts/deployDocs.sh]: Oops, yes. I added it.

**Comment by thuey** [comment]
@jonnymbgood Good point. I've got to run, but I'll do that when I get a chance.

**Comment by thuey** [comment]
@jonnymbgood Could you please review this again? Note, I will be pulling out www in a separate issue. With this change, the docs are now available on https://docs.pleaseops.com

**Pull request merged**